hack/tests/e2e-ansible.sh

```diff
@@ -145,8 +146,16 @@ pushd memcached-operator
 operator-sdk add crd --kind=Foo --api-version=ansible.example.com/v1alpha1
 sed -i 's|\(FROM quay.io/operator-framework/ansible-operator\)\(:.*\)\?|\1:dev|g' build/Dockerfile
 operator-sdk build "$DEST_IMAGE"
+# If using a kind cluster, load the image into all nodes.
+load_image_if_kind "$DEST_IMAGE"
 sed -i "s|{{ REPLACE_IMAGE }}|$DEST_IMAGE|g" deploy/operator.yaml
 sed -i 's|{{ pull_policy.default..Always.. }}|Never|g' deploy/operator.yaml
+# kind has an issue with certain image registries (ex. redhat's), so use a
+# different test pod image.
+METRICS_TEST_IMAGE="fedora:latest"
```

Comment:
Is loading images with the `:latest` tag okay?
https://kind.sigs.k8s.io/docs/user/quick-start/#loading-an-image-into-your-cluster
```
Note: The Kubernetes default pull policy is IfNotPresent unless the image tag is :latest in which case the default policy is Always. IfNotPresent causes the Kubelet to skip pulling an image if it already exists. If you want those images loaded into node to work as expected, please:

- don’t use a :latest tag
```

Seems like `:latest` would always pull down the image so I don't see what bug was being referred to here:
https://github.com/kubernetes-sigs/kind/issues/874#issuecomment-535710086

Plus it's effectively just a curl container so either way it shouldn't really matter if the image is updated.

Reply:
I can pin to `30` but I agree its not a big deal.